service-base/src/main/java/org/eclipse/hono/service/metric/MicrometerBasedMetrics.java

```diff
@@ -294,7 +294,7 @@ public final void reportTelemetry(
         // record payload size
         DistributionSummary.builder(METER_TELEMETRY_PAYLOAD)
             .baseUnit("bytes")
-            .minimumExpectedValue(0.0)
+            .minimumExpectedValue(0.01)
```

Comment:
what is the purpose of this?

Reply:
The value 0.01 is rather randomly chosen to avoid exception minimum expected value needs to be greater than zero.

Reply:
Ah, this seems to have changed in micrometer ...